services/catalog/src/simcore_service_catalog/core/application.py

```diff
@@ -67,6 +67,11 @@ def init_app(settings: ApplicationSettings | None = None) -> FastAPI:
     if app.state.settings.CATALOG_PROMETHEUS_INSTRUMENTATION_ENABLED:
         setup_prometheus_instrumentation(app)
 
+    if app.state.settings.CATALOG_PROFILING_MIDDLEWARE_ENABLED:
+        from servicelib.fastapi.profiler_middleware import ProfilerMiddleware
```

Comment:
why internal import?

Reply:
Yep, good point. This made sense back when I added it to the api-server, but no longer. So I have changed this.